DB/GADPerson.m

```diff
@@ -95,6 +101,50 @@ -(GADPerson *)initWithDictionary: (NSDictionary *) dict {
     return people;
 }
 
++ (void) fetchPersonInfoWithCriteria:(NSDictionary*)crit andUsername:(NSString*)usrname andPassword:(NSString*) pw completionHandler:(void(^_Nonnull)(NSArray<GADPerson *> *))completion {
+
+    NSMutableArray * queryItems = [NSMutableArray array];
+    for (NSString * key in crit){
+        [queryItems addObject:[NSURLQueryItem queryItemWithName:key value:crit[key]]];
+    }
+    NSURLComponents *components = [NSURLComponents componentsWithString:@"https://itwebappstest.grinnell.edu/DotNet/WebServices/api/db"];
+    components.queryItems=queryItems;
+    NSURL *url = components.URL;
+    
+    NSString *Post = [NSString stringWithFormat:@"{'un':'%@', 'pw':'%@'}",usrname,pw];
+    NSData *PostData = [Post dataUsingEncoding:NSUTF8StringEncoding];
+    NSString *PostLength = [NSString stringWithFormat:@"%lu",(unsigned long)[PostData length]];
+    
+    NSMutableURLRequest *req = [NSMutableURLRequest requestWithURL:url cachePolicy:NSURLRequestUseProtocolCachePolicy timeoutInterval:60.0];
```

Comment:
Can we make the timeout a constant at the top too? I forget what time `timeoutInterval` takes, but we'd declare `const <type> timeoutInterval = 60.0;` up with the `apiURL`.